README.md

```diff
@@ -18,11 +19,11 @@ Transparently use [Twitter Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
 - FormHelper (align: default, inline, horizontal)
 - HtmlHelper (components: breadcrumbs, badge, label, icon)
 - PaginatorHelper
-- Widgets (basic, checkbox, radio, button)
+- Widgets (basic, checkbox, radio, button, select, textarea)
```

Comment:
Can you please remove "checbox" form the list. It's deprecated and no longer used.